docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -78,7 +78,6 @@ plugins directory
 ------------------
 
 Add a 'per plugin type' specific subdirectory here, including ``module_utils`` which is usable not only by modules, but by most plugins by using their FQCN. This is a way to distribute modules, lookups, filters, and so on, without having to import a role in every play.
-
```

Comment:
Is this necessary?

Reply:
I didn't see the dash in the latest version. Please verify I'm not missing anything.

Reply:
Dash? It's a paragraph separator removal, right?

Reply:
This change is removing an empty line between the two paragraphs.